### PR TITLE
zopfli: move $prefix/include to $includeDir

### PR DESCRIPTION
**Pull request by Ra-V2**
https://github.com/haikuports/haikuports/pull/4560

### CODE REVIEW

**Comment by Begasus** [comment]
You can use cmake options to install the libraries and headers without the need to move them, also, you can create shared libs if you turn that option on
You'll need to provide the libraries and add a devel package then (also use the libVersion variant)

**Comment by Ra-V2** [comment]
@Begasus I did something like this. But I have no idea where are these options to move include in cmake.

**Comment by korli** [comment]
please bump the revision
where is the actual move of $prefix/include to $includeDir ?

**Comment by Ra-V2** [comment]
@korli I bump this revision and added again this move to recipe. No policy errors are currently popping up.

File: app-arch/zopfli/zopfli-1.0.3.recipe [hunk 1]
```diff
@@ -19,35 +19,53 @@ if [ "$targetArchitecture" = x86_gcc2 ]; then
...
 	"
 
 BUILD()
 {
 	mkdir -p build && cd build
 	cmake .. \
 		-DCMAKE_BUILD_TYPE=Release \
-		-DCMAKE_INSTALL_PREFIX=$prefix
+		-DCMAKE_INSTALL_PREFIX=$prefix \
+		-DZOPFLI_BUILD_SHARED=ON
```
Comment (Begasus) [inline app-arch/zopfli/zopfli-1.0.3.recipe]: Check CMakeCache.txt for the variables you can use for LIBDIR and INCLUDEDIR

File: app-arch/zopfli/zopfli-1.0.3.recipe [hunk 2]
```diff
@@ -19,35 +19,53 @@ if [ "$targetArchitecture" = x86_gcc2 ]; then
...
 	make $jobArgs
 }
 
 INSTALL()
 {
 	cd build
 	make install
+	mkdir -p $(dirname $includeDir)
+	mv $prefix/include $includeDir
```
Comment (Begasus) [inline app-arch/zopfli/zopfli-1.0.3.recipe]: No need for them:
```diff
-	mkdir -p $(dirname $includeDir)
-	mv $prefix/include $includeDir
```

**Comment by Ra-V2** [comment]
@Begasus Ok, I got this. I was checking these settings only with -L and I don't see all of them :)

**Comment by Begasus** [comment]
@Ra-V2 the option -L doesn't always reveal the needed options, but it's where you start, thanks for your contribution!

File: app-arch/zopfli/zopfli-1.0.3.recipe [hunk 3]
```diff
@@ -19,30 +19,48 @@ if [ "$targetArchitecture" = x86_gcc2 ]; then
...
+	"
+
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	"
 BUILD_PREREQUIRES="
 	cmd:cmake
 	cmd:gcc$secondaryArchSuffix
 	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
```
Comment (korli) [inline app-arch/zopfli/zopfli-1.0.3.recipe]: This is probably not needed.

**Pull request merged**